revscoring/errors.py

```diff
@@ -75,6 +75,12 @@ def __init__(self, datasources, info=None, arg=None):
         super().__init__("Query failed ({0}:{1})"
                          .format(datasources, info))
 
+class UnexpectedContentType(DependencyError):
+    def __init__(self, text, content_type, arg=None):
+        super().__init__("Expected content of type {}, "
+                         "but the following can't be parsed "
+                         "(max 50 chars showed): {}"
```

Comment:
```suggestion
                         "(max 50 chars showed): {1}"
```

Reply:
Definitely I'll add the change to my patch!